tests/roles/development_environment/files/pre_launch.bash

```diff
@@ -118,7 +118,14 @@ ${BASH_ALIASES[openstack]} security group rule list --protocol tcp --ingress -f
 
 export FIP=192.168.122.20
 # check connectivity via FIP
-ping -c4 ${FIP}
```

Comment:
this will not work as set -e is set on top so script will exit on first failure. we can add set +e before ping block, and set -e post it.